src/server.py

```diff
@@ -411,6 +428,29 @@ def _ensure_release_countdown(self):
         self._countdown_id = GLib.timeout_add_seconds(
             self._TIMEOUT_S, self.release, priority=GLib.PRIORITY_LOW)
 
+    def _priority_playing(self):
+        for uuid_ in self.players:
+            if self.players[uuid_].priority:
+                return True
+        return False
```

Comment:
Probably not necessary, but this can be pythonized with `any`:

```
return any(player.priority for player in self.players.itervalues())
```

Reply:
Side from being more python'eske is there any other benefit? I am curious.

Reply:
Doesn't that iterate twice in the worst case? Not sure

Reply:
It's shorter and it's not hard to read. And no, it wouldn't cause double iteration since the elements of the generator are generated on demand.